pkg/config/setup/system_probe.go

```diff
@@ -120,7 +120,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Setup) {
 	// settings for system-probe in general
 	cfg.BindEnvAndSetDefault(join(spNS, "enabled"), false, "DD_SYSTEM_PROBE_ENABLED")
 	cfg.BindEnvAndSetDefault(join(spNS, "external"), false, "DD_SYSTEM_PROBE_EXTERNAL")
-	cfg.SetKnown(join(spNS, "adjusted")) //nolint:forbidigo // TODO: replace by 'SetDefaultAndBindEnv'
+	cfg.SetKnown(join(spNS, "adjusted")) //nolint:forbidigo // runtime setting only
```

Comment:
Let set a default to false since we're checking the source in the code. With Go it should change nothing since it's the default return for `GetBool`.

Reply:
Done.